src/modules/menu/components/share/MenuShareEmbed.vue

```diff
@@ -1,25 +1,34 @@
 <template>
     <div class="menu-share-embed">
-        <button class="btn btn-light embedded-button" @click="toggleEmbedSharing">
-            <FontAwesomeIcon icon="plus" />
+        <button
+            class="btn btn-light embedded-button"
+            data-cy="menu-share-embed-button"
+            @click="toggleEmbedSharing"
+        >
```

Comment:
Maybe use a small button `btn-sm` here. The `+ Embed` is bigger than the menu title `Share`

Reply:
Instead of a `+` `-` icons I would use the arrow right and down icon like in the menu and drop down buttons.